dubbo-common/src/main/java/org/apache/dubbo/common/utils/StringUtils.java

```diff
@@ -343,6 +343,20 @@ public static boolean isBlank(String str) {
         return isEmpty(str);
     }
 
+    public static boolean isRealBlank(CharSequence cs) {
+        int strLen;
+        if (cs != null && (strLen = cs.length()) != 0) {
+            for (int i = 0; i < strLen; ++i) {
+                if (!Character.isWhitespace(cs.charAt(i))) {
+                    return false;
+                }
+            }
+            return true;

```

Comment:
I think this return (after for loop)is not requires and remove the else and keep just `return true;`